lib/tasks/projects.rake

```diff
@@ -313,17 +313,14 @@ namespace :projects do
     end
 
     batch_size = 500
-    batch_wait = 5
 
     result_count = 0
 
     Project.platform(platform).find_in_batches(batch_size: batch_size).each do |projects|
-      if commit
-        projects.each(&:manual_sync)
-        sleep batch_wait
```

Comment:
If line 322 is just adding jobs to the queue, I don't think we need to stagger that since it's just putting stuff onto the Sidekiq queue, which gets run when resources become available.